var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -246,6 +253,30 @@ def check_compilers(self):
                 + " has no Fortran compiler added in spack. Add it or use openblas~fortran!"
             )
 
+    @property
+    def headers(self):
+        # The only public headers for cblas and lapacke in
+        # openblas are cblas.h and lapacke.h. The remaining headers are private
+        # headers either included in one of these two headers, or included in
+        # one of the source files implementing functions declared in these
+        # headers.
+        return find_headers(["cblas", "lapacke"], self.prefix.include)
+
+    @property
+    def libs(self):
+        spec = self.spec
+
+        # Look for openblas{symbol_suffix}
+        name = ["libopenblas", "openblas.lib"]
```

Comment:
This should be:

```
name = ["libopenblas", "openblas"]
```

(`find_libraries` just wants the name, and will append `.lib` itself).